src/pj_observation.c

```diff
@@ -112,3 +168,25 @@ int pj_show_triplet (FILE *stream, const char *banner, PJ_TRIPLET point) {
         i += fprintf(stream, "\n");
     return i;
 }
+
+PJ *pj_pj (PJ_CONTEXT *ctx, char *definition) {
+    if (0==ctx)
+        return pj_init_plus_ctx (pj_get_default_ctx(), definition);
+    return pj_init_plus_ctx (ctx, definition);
+}
+
+PJ *pj_pj_argv (PJ_CONTEXT *ctx, int argc, char **argv) {
+    return pj_init_ctx (ctx, argc, argv);
+}
+
+PJ_CONTEXT *pj_ctx (PJ *P) {
+    if (0==P)
+        return pj_get_default_ctx ();
+    return pj_get_ctx (P);
+}
+
+void pj_ctx_set (PJ *P, PJ_CONTEXT *ctx) {
```

Comment:
Probably reason closely related to the above mentioned.  Basically I have never seen it in use.

This might be another of the "drop them now, and reintroduce if necessary later" crowd